File: Conditional_Python/09_sky_trip.py
nights = int(input()) - 1
type_of_room = input()
qualification = input()
price_per_night = 0
total_money = 0

if type_of_room == "room for one person":
    price_per_night = 18
    total_money = price_per_night * nights
    if qualification == "positive":
        total_money *= 1.25
    elif qualification == "negative":
        total_money *= 0.9
    print(f"{total_money:.2f}")

elif type_of_room == "apartment":
    price_per_night = 25
    total_money = price_per_night * nights
    if nights < 10:
        total_money *= 0.7
    elif 10 <= nights <= 15:
        total_money *= 0.65
    elif nights > 15:
        total_money *= 0.5

    if qualification == "positive":
        total_money *= 1.25
    elif qualification == "negative":
        total_money *= 0.9
    print(f"{total_money:.2f}")

elif type_of_room == "president apartment":
    price_per_night = 35
    total_money = price_per_night * nights
    if nights < 10:
        total_money *= 0.9
    elif 10 <= nights <= 15:
        total_money *= 0.85
    elif nights > 15:
        total_money *= 0.8

    if qualification == "positive":
        total_money *= 1.25
    elif qualification == "negative":
        total_money *= 0.9
    print(f"{total_money:.2f}")

File: Conditional-JavaScript/09.skyTrip.js
function skyTrip(input){
    let nights = Number(input[0]) - 1;
    let typeOfRoom = input[1];
    let opinion = input[2];
    let price = 0;
    let tripCost = 0;

    if (typeOfRoom === "room for one person"){
        price = 18;
        if (nights < 10){
            tripCost = nights * price;
        } else if (nights >= 10 && nights <=15){
            tripCost = nights * price;
        } else if (nights > 15){
            tripCost = nights * price;
        }

    } else if (typeOfRoom === "apartment"){
        price = 25;
        if (nights < 10){
            tripCost = nights * price * 0.7;
        } else if (nights >= 10 && nights <=15){
            tripCost = nights * price * 0.65;
        } else if (nights > 15){
            tripCost = nights * price * 0.5;
        }

    } else if (typeOfRoom === "president apartment"){
        price = 35;
        if (nights < 10){
            tripCost = nights * price * 0.9;
        } else if (nights >= 10 && nights <=15){
            tripCost = nights * price * 0.85;
        } else if (nights > 15){
            tripCost = nights * price * 0.8;
        }
    }

    if (opinion === "positive"){
        tripCost *= 1.25;
        console.log((tripCost).toFixed(2))
    } else if(opinion === "negative"){
        tripCost *= 0.9;
        console.log((tripCost).toFixed(2))
    }
}
//skyTrip(["14", "apartment", "positive"])
//skyTrip(["30", "president apartment", "negative"])
//skyTrip(["12", "room for one person", "positive"])
//skyTrip(["2", "apartment", "positive"])